pkg/kubelet/prober/worker.go

```diff
@@ -62,6 +63,11 @@ type worker struct {
 	resultRun int
 }
 
+const (
+	// jitter factor for worker.doProbe sync loop
+	workerProbeJitterFactor = 1.0
```

Comment:
I'm worried this factor is too large. The probe period is user-configured, so I think this should be considered a user-facing change. At the very least, this needs to be documented in the probe API.